src/styles/css/main.css

```
@@ -0,0 +1,345 @@
+            format('embedded-opentype'),
+        url('../../assets/fonts/icomoon.ttf?wl83df') format('truetype'),
+        url('../../assets/fonts/icomoon.woff?wl83df') format('woff'),
+        url('../../assets/fonts/icomoon.svg?wl83df#icomoon') format('svg');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+[class^='icon-'],
+[class*=' icon-'] {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    font-family: 'icomoon' !important;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)